OpenGraph/functions/graph_embedding/node2vec.py

```diff
@@ -3,14 +3,17 @@
 import OpenGraph as og
```

Comment:
maybe we can use the "tqdm" library to check the progress :D

Reply:
OK